BackendNew/scripts/create-tables.sql

```diff
@@ -3,27 +3,26 @@
  Vacancy table, Student table, Student_Select_Company table, Monthly report table and Feedback table
  */
 CREATE TABLE Role (
-    RoleId int NOT NULL AUTO_INCREMENT,
     Name varchar(255) NOT NULL,
     Description varchar(255),
-    PRIMARY KEY (RoleId)
+    PRIMARY KEY (Name)
 );
 
 CREATE TABLE Admin (
     AdminId int NOT NULL AUTO_INCREMENT,
-    RoleId int,
+    RoleName int,
     Name varchar(255) NOT NULL,
     PhoneNumber int,
     Email varchar(255),
```

Comment:
Can you Email NOT NULL

Reply:
It depends. But the admin id should be filled

Reply:
So in case if password is forgotten

Reply:
Okay I will make it NOT NULL